recipes/newrelic/infrastructure/logs/logs.yml

```diff
@@ -126,15 +124,28 @@ install:
             done
           fi
 
+          discoveredLogFiles=$(echo -n "{{.DISCOVERED_LOG_FILES | trim}}")
+          discoveredLogsStringLength=${#discoveredLogFiles}
+
           # If the user provides log files that don't exist and we haven't discovered any log files,
           # we cannot proceed with the installation.
           if [ $foundLogFilesCount -eq 0 ] && { [ $discoveredLogsStringLength -eq 0 ] || [ "$discoveredLogFiles" = "logs: []" ]; }; then
             echo "ERROR Provided log files do not exist: $logFiles. No log file(s) found or discovered. Cannot proceed. Exiting installation." >> /dev/stderr
             exit 16
           fi
 
+          # Build the discovered.yml logs config file used for Docker logs
+          IFS=', ' read -r discoveredLogFilesArray <<< "$discoveredLogFiles"
+
           if [ $discoveredLogsStringLength -gt 0 ]; then
-            echo -e "{{.DISCOVERED_LOG_FILES}}" | sudo tee /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
+            echo "logs:" | sudo tee /etc/newrelic-infra/logging.d/discovered.yml > /dev/null
+
+            for filePath in "${discoveredLogFilesArray[@]}"
+            do
+              if [ -f $filePath ]; then
```

Comment:
I dont think we should validate filepath here.
We just need to write the config, the path may exist later and the agent will pick it up, that's the main case actually for docker logs which are generated in a guid sub-folder

Reply:
Makes sense. I'll update.